core/src/mindustry/ui/fragments/ChatFragment.java

```diff
@@ -62,9 +62,8 @@ public ChatFragment(){
 
         update(() -> {
 
-            if(net.active() && input.keyTap(Binding.chat) && (scene.getKeyboardFocus() == chatfield || scene.getKeyboardFocus() == null)){
+            if(net.active() && input.keyTap(Binding.chat) && (scene.getKeyboardFocus() == chatfield || scene.getKeyboardFocus() == null || ui.minimapfrag.shown()))
                 toggle();
-            }
```

Comment:
Don't remove the brace here. There's no point to making a braceless if-statement if it's on two different lines anyway.